yaml/landuse.yaml

```diff
@@ -1085,7 +1085,7 @@ filters:
     min_zoom: { clamp: { max: 16, min: 9, value: { col: zoom } } }
     output:
       <<: *output_properties
-      kind: danger
+      kind: danger_area
```

Comment:
That's a better kind, thanks! Sometimes we'd say `military_danger_area` but I don't think there are other types of danger areas so this is good as you propose.